source/__tests__/integration/export-profile.test.js

```diff
@@ -27,6 +27,7 @@ describe('Sinopia Profile Editor exports a loaded Profile', () => {
       await page.waitFor(1000)
       await expect(page).toClick('a', { text: 'Export'})
       // TODO: test that profile written to file system (see #77)
+      // TODO: test fields that are validated, listed in source/assets/js/modules/profile/services/profileHandler.service.js
```

Comment:
could you please add a ticket to github for this?  Might even be a thing worth doing while you're in DC?

Reply:
I created #122. I will see if I can get to modifying the test for this while I'm in DC. I will make a separate ticket about that.

Reply:
See #123